README.md

```diff
@@ -2,34 +2,29 @@
 [![Build Status](https://travis-ci.org/gfx-rs/portability.svg?branch=master)](https://travis-ci.org/gfx-rs/portability)
 [![Gitter](https://badges.gitter.im/gfx-rs/portability.svg)](https://gitter.im/gfx-rs/portability)
 
-This is a prototype library implementing [Vulkan Portability Initiative](https://www.khronos.org/blog/khronos-announces-the-vulkan-portability-initiative) using gfx-rs [low-level core](http://gfx-rs.github.io/2017/07/24/low-level.html). See gfx-rs [meta issue](https://github.com/gfx-rs/gfx/issues/1354) for backend limitations and further details.
+This is a prototype library implementing [Vulkan Portability Initiative](https://www.khronos.org/blog/khronos-announces-the-vulkan-portability-initiative) using [gfx-hal](http://gfx-rs.github.io/2017/07/24/low-level.html). See gfx-rs [meta issue](https://github.com/gfx-rs/gfx/issues/1354) for backend limitations and further details.
 
-## Check out
-```
-git clone --recursive https://github.com/gfx-rs/portability && cd portability
-```
+## Showcase
 
-## Build
+[Dota2](https://github.com/ValveSoftware/Dota-2):
+![Dota2](etc/dota2-river.jpg)
 
-### Makefile (Unix)
-```
-make
-```
+[RPCS3](https://github.com/RPCS3/rpcs3):
+![RPCS3-cube](etc/rpcs3-cube.png)
+![RPCS3-scogger](etc/rpcs3-scogger.png)
 
-### CMake (Window)
-Build the Rust library (portability implementation):
+[Dolphin](https://github.com/dolphin-emu):
+![Dolphin-sb](etc/dolphin-smash-bros.png)
+![Dolphin-pm](etc/dolphin-paper-mario.png)
 
-```
-cargo build --manifest-path libportability/Cargo.toml --features <vulkan|dx12|metal>
-```
+## Instructions
 
-Build the native example:
+Despite the fact it's written in Rust, the produced binary is a standard dynamic library that can be linked to from any program (written in the language of your choice). Typically, you'd need to create a symbolic link with a name that a target application expects, e.g. `libvulkan.dylib -> libportability.dylib`.
 
+Check out and build:
 ```
-mkdir build
-cd build
-cmake ..
-cmake --build . --target native_test
+git clone --recursive https://github.com/gfx-rs/portability && cd portability
+cargo build --manifest-path libportability/Cargo.toml --features <vulkan|dx12|metal>
```

Comment:
The only downside with this is that this misses the version setting, so it doesn't work with Dota2 at least (maybe others?)

Reply:
I think if people want to build it themselves, they should contact us. But yes, let's follow up with some more detailed instructions.